spec/extensions/linear_algebra_functions.md

```diff
@@ -270,7 +270,7 @@ Computes the multiplicative inverse of a square matrix (or a stack of square mat
 (function-linalg-lstsq)=
 ### linalg.lstsq(x1, x2, /, *, rtol=None)
 
-Returns the least-squares solution to a linear matrix equation `Ax = b`.
+Returns the minimum-norm least-squares solution to a linear matrix equation `Ax = b`.
```

Comment:
Two questions:
1. Following the same argument that led to the other changes in this PR, shouldn't we say "...*a* minimum-norm least-squares solution..." here? Is the solution always unique? 
2. English problem: Doesn't "least-squares" already refer to the L2 norm? So maybe we should just say "... a minimal least-squares solution..."?

Reply:
The minimum norm solution is indeed always unique. It is given by `A^{\dagger}b`, where `A^{\dagger}` is the pseudoinverse of `A` (see the [wiki](https://en.wikipedia.org/wiki/Moore%E2%80%93Penrose_inverse#Applications)).

While least squares most of the times refers to the L2 norm, I think one still needs to explicit that is minimum with respect to a norm. I have never seen the expression "minimum vector".

Reply:
This API has been removed from the spec. Will mark as resolved.